drizzlepac/hlautils/catalog_utils.py

```diff
@@ -824,6 +825,13 @@ def __init__(self, image, param_dict, param_dict_qc, diagnostic_mode, tp_sources
         self._border = self.param_dict["sourcex"]["border"]
         self._nsigma = self.param_dict["nsigma"]
 
+        # Columns to include from the computation of source properties to save
+        # computation time from computing values which are not used
+        self.include_filter_cols = ['background_at_centroid', 'bbox_xmax', 'bbox_xmin', 'bbox_ymax', 'bbox_ymin',
```

Comment:
This list seems a little too spartan.  Properties such as 'area' hold obvious use for the end-user and correspond to a column provided by the HLA catalog.  Granted, the point of this was to limit the computations, but we also do not want to end up leaving out information valuable for the end-user.  I would suggest that this list be reviewed by key stakeholders prior to merging to insure the proper balance.

Reply:
I only used the values generated by Photutils which look like they correspond to the values generated in the Source Extractor catalog for the HLA project.  Note there are additional columns which contain photometry data as produced by the photometry_tools.iraf_style_photometry().  I will ask users for their input.

Reply:
Email requesting input on further values to be included in the Segmentation catalog has been sent to: Jenn, Varun, Roberto, Matt, Hien, Rick, and Steve.

Reply:
Until more requests, added "area" and "source_sum_error" (aka FluxIsoError).